canonical-web-and-design/vanilla-framework
# Update notification spacing

## Done
Fixes #3865 
Improve notification spacing.
Driveby: unhardcode a font-weight declaration, replace a value with an existign variable for that value ($input-margin-bottom)

## QA

- Open [demo](https://vanilla-framework-3866.demos.haus/docs/examples)
- Check against baseline grid
- Review updated documentation:
  - added Inset variant to reflect conversation with @cassiocassio and @long-chung about the use case described in gov.uk inset notifications (when directly related to page content, as opposed to notifying the user about a system status that may not necessarily be related to what they are doing at the moment). The ux guidance on this will need some wordsmithing.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
![image](https://user-images.githubusercontent.com/2741678/125490087-5c9ef8b8-daba-4a06-9936-4a1627a9488b.png)
